CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Versions suffixed with `b*` are in `beta` and can be installed with `pip install --pre betterproto`.
 
+## [2.0.0b6] - 2023-06-05
+
+- Added support for @generated marker [#382](https://github.com/danielgtaylor/python-betterproto/pull/382)
+- Pull down the include_default_values argument to to_json [#405](https://github.com/danielgtaylor/python-betterproto/pull/405)
+- Fix typechecker compatiblity checks in server streaming methods [#413](https://github.com/danielgtaylor/python-betterproto/pull/413)
+- Enforce serialize_empty for repeated fields [#417](https://github.com/danielgtaylor/python-betterproto/pull/417)
+- Pythonize input_type name in py_input_message [#436](https://github.com/danielgtaylor/python-betterproto/pull/436)
+- Drop support for python3.6 [#444](https://github.com/danielgtaylor/python-betterproto/pull/444)
+- Fix CI [#456](https://github.com/danielgtaylor/python-betterproto/pull/456)
+- Add support for pydantic dataclasses [#406](https://github.com/danielgtaylor/python-betterproto/pull/406)
+- Fix for [#459](https://github.com/danielgtaylor/python-betterproto/issues/459) (pydantic code gen only) [#460](https://github.com/danielgtaylor/python-betterproto/pull/460)
```

Comment:
I don't particularly think this is worth mentioning considering it fixes something that was never publicly released but if you want to leave it in that's fine